### ISSUE TITLE
TabHostView does not update background color in dark mode

### ISSUE THREAD

**Issue by jbachelor** [roubachof/Sharpnado.Presentation.Forms#94]
**First and Foremost:**
Thank you so much for this awesome library... I sincerely appreciate all the hard work you've put into it!

**Platform (please complete the following information):**
 - OS: Both
 - Device: iPhone 11 simulator running 13.2; Android Pixel 2 emulator running 10.0/API 29
 - Xamarin.Forms: [4.2.0.815419 and 4.4.0.991265]

**Describe the bug**
In my efforts to implement Dark Mode support for Android and iOS along with a Sharpnado tab bar, I've found that the tab bar does not seem to update its color after it is initially set, while other components do update. 

**To Reproduce -- Short version**
1. Grab my modified [Sharpnado sample here](https://drive.google.com/open?id=1U9xFai_SNLO1BZ2_gUHfqRB0gv8uGx8I).
2. When you run the app on an iOS or Android device that supports dark mode, try switching from light to dark. Notice that the ContentPage will update to a new color, but the tab bar remains in the original light color.
    * See [this gif](https://drive.google.com/open?id=1NE9xcUc7iZwVsA9oga2NOUIMYyYODjI0) for a look at a demo on an iOS simulator running iOS 13.2.

**The long version of how to reproduce:**
In my sample project (link below), I'm using the standard Sharpnado sample app with the following modifications:
* Added a dependency service to find out whether the phone is in light or dark mode (IDeviceThemeService, DeviceThemeService_Android, DeviceThemeService_iOS).
* Added code to App.xaml.cs to set the app theme from OnStart and OnResume (SetAppThemeAsync).
* Moved all styles from App.xaml to Styles/ThemeIndependent.xaml.
* Added new ResourceDictionaries called "Styles/LightMode.xaml" and "Styles/DarkMode.xaml". Each of these merges with the ThemeIndependent ResourceDictionary.
* Added one color to the DarkMode and LightMode ResourceDictionaries (key="TabBarBackground").
* Set the BackgroundColor properties of the ContentPage itself and the TabHostView both to be set to "TabBarBackground" as a DynamicResource.
* Run the app on a device (iOS or Android) that supports dark mode. Notice that the ContentPage will change color depending on light/dark mode of the phone, but the tab bar will not.

**Exceptions (if applicable)**
n/a

**Screenshots (if applicable)**
* Setting background colors of ContentPage and TabHostView to the same dynamic resource:

![image](https://user-images.githubusercontent.com/19045/71120783-f4d81d80-2191-11ea-8d90-b469ddf38257.png)

* Light mode:  Both ContentPage and TabHostView are light-yellow.
![image](https://user-images.githubusercontent.com/19045/71120954-44b6e480-2192-11ea-8fbd-283775ca7ec8.png)

* Dark mode:  Only the ContentPage gets the updated color.
![image](https://user-images.githubusercontent.com/19045/71120994-5bf5d200-2192-11ea-9d34-243cde7271c7.png)

**Comment by roubachof** [comment]
man you're my favorite bug reporter :)

**Comment by jbachelor** [comment]
I figure it's the least I can do... I am super grateful for this awesome library!  =D

**Comment by roubachof** [comment]
ahah found it! Will be fixed in 1.4.

**Comment by muhaym** [comment]
Hi @roubachof  Seems this affects other Colour properties as well. 
SelectedTabColor, UnselectedIconColor, UnselectedLabelColor - Need to Exit App and open to see the changes reflected properly

Settings in this way
```
  <ContentPage.Resources>
        <ResourceDictionary>
            <converter:NameToShortConverter x:Key="NameToShortConverter" />
            <Style x:Key="BottomTabStyle" TargetType="tabs:BottomTabItem">
                <Setter Property="SelectedTabColor" Value="{DynamicResource IconPrimaryColor}" />
                <Setter Property="UnselectedIconColor" Value="{DynamicResource IconColor}" />
                <Setter Property="UnselectedLabelColor" Value="{DynamicResource IconColor}" />
                <Setter Property="LabelSize" Value="14" />
                <Setter Property="IconSize" Value="28" />
                <Setter Property="IsTextVisible" Value="True" />
            </Style>
        </ResourceDictionary>
    </ContentPage.Resources>
```

```

            <tabs:TabHostView.Tabs>
                <tabs:BottomTabItem
                    x:Name="HomeTab"
                    IconImageSource="home.png"
                    Label="{x:Static resource:AppResources.hometab}"
                    Style="{StaticResource BottomTabStyle}" />
                <tabs:BottomTabItem
                    x:Name="MyCourseTab"
                    IconImageSource="courses.png"
                    Label="{x:Static resource:AppResources.mycourse}"
                    Style="{StaticResource BottomTabStyle}" />

            </tabs:TabHostView.Tabs>
```

Inconsistent behaviour during Theme Changes.


![ezgif com-optimize](https://user-images.githubusercontent.com/10905479/95031571-5727ed80-06d4-11eb-84e3-ed087e889117.gif)


Am I doing something wrong?
